README.md

```diff
@@ -7,9 +7,69 @@ See [Cap from the ORNL-CEES team](https://github.com/ORNL-CEES/Cap "Github - ORN
  +  graphical representation of sample output data generated from Dualfoil5.2
  +  tests to verify the functionality of the `battery` package
 
+## Installation
+ 1. Make sure you have downloaded the latest non-beta version of [docker](https://www.docker.com/products/overview "Get Docker")
+ 2. Pull the Cap image `dalg24/cap` by copying and pasting the following command into your shell:
+
+ ```
+ docker pull dalg24/cap
+ ```
+
+ 3. Now clone this repository with the following command:
+
+ ```
+ git clone https://github.com/iSchomer/Dualfoil_Storage_Device.git
+ ```
+
+ 4. This should download a `Dualfoil_Storage_Device` directory. Change into it.
+ 5. Lastly, run a docker container with the pycap image:
+
+ ```
+ docker run --rm -it -p 8888:8888 -v $PWD:/notebooks dalg24/cap
+ ```
+
+ 6. Pycap and this code can be accessed by opening up a browser and going to `http://<ip_address>:8888` where `ip_address` is the IP address of the machine with the Docker daemon running.
+
+Once you have opened into the Jupyter notebook, you will have access to pycap as well as all the code included in this repository.
+
+
 ## Basic Use
- 1. Dualfoil is an instance of pycap.EnergyStorageDevice
- 2. Dualfoil5.1 is run through and managed by the `Dualfoil` object, which is found in `energy_storage_device.py` of the `battery` package
- 3. The docker folder holds the dualfoil5.1 files; `Dualfoil` needs only a relative path to this folder to function properly
- 4. `Dualfoil` can run individual legs of a simulation, return output values, and graph those values in 3D
- 5. For more complicated simulations, a `Dualfoil` object can be passed into one of Pycap's experiments, such as `CyclicChargeDischarge`
+ + Dualfoil is an instance of pycap.EnergyStorageDevice
+ + `Dualfoil` can run individual legs of a simulation, return output values, and graph those values in 3D
+ + For more complicated simulations, a `Dualfoil` object can be passed into one of Pycap's experiments, such as `CyclicChargeDischarge`
+
+ Here is an example of some basic operations:
+ ```python
+ from pycap import PropertyTree, Charge
+ from battery import *
+ fom energy_storage_device import Dualfoil
+
+ # can run dualfoil sim manually 
+ device = Dualfoil(path='docker/')
+ print(device.get_voltage())
+ # charge for 1 minute
```

Comment:
The starting voltage for the cell is around 4.53V... what sounds like a more reasonable current to you?